lib/ae_mdw/collection.ex

```diff
@@ -54,10 +54,10 @@ defmodule AeMdw.Collection do
   @doc """
   Builds a stream from records from a table starting from the initial_key given.
   """
-  @spec stream(table(), direction(), scope(), cursor()) :: Enumerable.t()
-  def stream(tab, direction, scope, cursor) do
+  @spec stream(table(), direction(), key_boundary(), cursor()) :: Enumerable.t()
+  def stream(tab, direction, key_boundary, cursor) do
     {first, last} =
-      case {scope, direction} do
+      case {key_boundary, direction} do
```

Comment:
Love the renaming, we can now refer to scope in the context of generations/txis and boundaries in the context of database page boundaries!